components/tikv_alloc/Cargo.toml

```diff
@@ -42,3 +39,7 @@ features = ["stats"]
 [dependencies.jemalloc-ctl]
 version = "0.2.0"
 optional = true
+
+[dependencies.tcmalloc]
+version = "0.2.0"
```

Comment:
can we use @brson's branch so we can link tcmalloc statically?

Reply:
@siddontang 

```rust
[dependencies.tcmalloc]
# version = "0.2.0"
optional = true
git = "https://github.com/brson/tcmalloc-rs"
features = ["bundled"]
```
This should work, but only try in linux OS.  But It is recommended to make other attempts after the merge PR.

Reply:
I'll land that one after this does.